packages/pwa-kit-build/src/ssr/server/loading-screen/index.html

```diff
@@ -0,0 +1,342 @@
+        }
+
+        /* tablet */
+        @media (max-width: 1024px) {
+            .gutter {
+                max-width: 32px;
+                min-width: 32px;
+            }
+            .message h1 {
+                margin-bottom: 10px;
+                font-size: 30px;
+                line-height: 38px;
+            }
+            .message p {
+                margin-bottom: 12px;
+                font-size: 16px;
+                line-height: 24px;
+            }
+        }
+
+        /* mobile, small tablet */
+        @media (max-width: 600px) {
+
+            .gutter {
+                display: none;
+            }
+            main {
+                flex-direction: column-reverse;
+                justify-content: center;
+                align-items: normal;
+                padding: 40px;
+            }
+            .message, .codey {
+                flex-basis: auto;
+            }
+            .message {
+                text-align: center;
+            }
+            .message h1 {
+                margin-bottom: 10px;
+                font-size: 24px;
+                line-height: 30px;
+            }
+            .message p {
+                margin-bottom: 6px;
+                font-size: 14px;
+                line-height: 21px;
+            }
+
+            .external-links svg {
+                margin-right: 0;
+                width: 32px;
+                height: 32px;
+            }
+            .external-links li {
+                margin-right: 24px;
+            }
+            .external-links li:last-child {
+                margin-right: 0;
+            }
+            .external-links span {
+                display: none;
+            }
+        }
+    </style>
+</head>
+<body>
+    <main>
+        <div class="gutter"></div>
+        <div class="message">
+            <div class="js-rotating-headings" data-interval-in-seconds="5">
+                <h1>Building your app...</h1>
+                <h1>Work, work, work</h1>
+                <h1>Build, build, build</h1>
+            </div>
+
+            <div class="js-random-fun-facts" data-interval-in-seconds="30">
+                <p>Fun Fact: The historical uptime with PWA Kit and MRT is 99.99%.</p>
+                <p>Fun Fact: Maximizing your cache hit ratio can drastically improve storefront performance. <a href="https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/maximizing-your-cache-hit-ratio.html">Learn more.</a></p>
+                <p>Fun Fact: In PWA Kit 1.4, we now support React 17. <a href="https://github.com/SalesforceCommerceCloud/pwa-kit/releases">Learn more.</a></p>
+                <p>Fun Fact: AppConfig is one of 4 special components used to add functionality across multiple pages. <a href="https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/special-components.html">Learn more.</a></p>
+            </div>
+
+            <div hidden class="js-error-message" data-show-after-seconds="60">
```

Comment:
We must not ship with this. This code waits for 60s before _guessing_ that there is something wrong with the build, based on no evidence whatsoever. We must not guess here: We either detect an actual error and show an appropriate screen, or we should do nothing. I gave this feedback earlier on in a design review!

Reply:
Removing this timeout for now